Add missing comas in sql statement (#1219)
Fixed by upstream commit a9c6748ec7831ba5e0323186f148edb78ba9aa05
--- a/src/nodetree_tree_sql.erl
+++ b/src/nodetree_tree_sql.erl
@@ -77,9 +77,9 @@
 	    catch
 	    ejabberd_sql:sql_query_t(
               ?SQL("update pubsub_node set"
-                   " host=%(H)s"
-                   " node=%(Node)s"
-                   " parent=%(Parent)s"
+                   " host=%(H)s,"
+                   " node=%(Node)s,"
+                   " parent=%(Parent)s,"
                    " type=%(Type)s "
                    "where nodeid=%(OldNidx)d")),
 	    OldNidx;
